tests/roles/backend_services/templates/openstack_version.j2

```diff
@@ -0,0 +1,74 @@
+    mariadbImage: {{ container_registry }}/{{ container_namespace }}/openstack-mariadb:{{ container_tag }}
+    neutronAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-neutron-server:{{ container_tag }}
+    novaAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-nova-api:{{ container_tag }}
+    novaComputeImage: {{ container_registry }}/{{ container_namespace }}/openstack-nova-compute:{{ container_tag }}
+    novaConductorImage: {{ container_registry }}/{{ container_namespace }}/openstack-nova-conductor:{{ container_tag }}
+    novaNovncImage: {{ container_registry }}/{{ container_namespace }}/openstack-nova-novncproxy:{{ container_tag }}
+    novaSchedulerImage: {{ container_registry }}/{{ container_namespace }}/openstack-nova-scheduler:{{ container_tag }}
+    octaviaAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-octavia-api:{{ container_tag }}
+    octaviaHealthmanagerImage: {{ container_registry }}/{{ container_namespace }}/openstack-octavia-health-manager:{{ container_tag }}
+    octaviaHousekeepingImage: {{ container_registry }}/{{ container_namespace }}/openstack-octavia-housekeeping:{{ container_tag }}
+    octaviaWorkerImage: {{ container_registry }}/{{ container_namespace }}/openstack-octavia-worker:{{ container_tag }}
+    openstackClientImage: {{ container_registry }}/{{ container_namespace }}/openstack-openstackclient:{{ container_tag }}
+    osContainerImage: {{ container_registry }}/{{ container_namespace }}/edpm-hardened-uefi:{{ container_tag }}
+    ovnControllerImage: {{ container_registry }}/{{ container_namespace }}/openstack-ovn-controller:{{ container_tag }}
+    ovnControllerOvsImage: {{ container_registry }}/{{ container_namespace }}/openstack-ovn-base:{{ container_tag }}
+    ovnNbDbclusterImage: {{ container_registry }}/{{ container_namespace }}/openstack-ovn-nb-db-server:{{ container_tag }}
+    ovnNorthdImage: {{ container_registry }}/{{ container_namespace }}/openstack-ovn-northd:{{ container_tag }}
+    ovnSbDbclusterImage: {{ container_registry }}/{{ container_namespace }}/openstack-ovn-sb-db-server:{{ container_tag }}
+    placementAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-placement-api:{{ container_tag }}
+    rabbitmqImage: {{ container_registry }}/{{ container_namespace }}/openstack-rabbitmq:{{ container_tag }}
+    swiftAccountImage: {{ container_registry }}/{{ container_namespace }}/openstack-swift-account:{{ container_tag }}
+    swiftContainerImage: {{ container_registry }}/{{ container_namespace }}/openstack-swift-container:{{ container_tag }}
+    swiftObjectImage: {{ container_registry }}/{{ container_namespace }}/openstack-swift-object:{{ container_tag }}
+    swiftProxyImage: {{ container_registry }}/{{ container_namespace }}/openstack-swift-proxy-server:{{ container_tag }}
+    cinderVolumeImages:
+      default: {{ container_registry }}/{{ container_namespace }}/openstack-cinder-volume:{{ container_tag }}
```

Comment:
cinder and manila-share still have an issue - otherwise this is good \o/ 

https://logserver.rdoproject.org/95/52995/2/check/periodic-data-plane-adoption-osp-17-to-extracted-crc/f7c0a71/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/pods/cinder-volume-volume1-0/cinder-volume-volume1-0-describe

```
        *     Image:         quay.io/podified-antelope-centos9/openstack-cinder-volume@sha256:886942a49a89be8ee12b3b6c220cbea3aead4ee3d9f18f97757eb7d795f241c6 
```

and https://logserver.rdoproject.org/95/52995/2/check/periodic-data-plane-adoption-osp-17-to-extracted-crc/f7c0a71/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/pods/manila-share-share1-0/manila-share-share1-0-describe

```
        *     Image:         quay.io/podified-antelope-centos9/openstack-manila-share@sha256:4bc9f7d690a53f690e6ac27eedc78d1e9d8bcb9338fad89f9d3e2661de44c3dd 
```

Reply:
@dprince o/ hello any idea what I'm missing for those two please? ^^ 

the OpenStackVersion we end up with is there https://logserver.rdoproject.org/95/52995/2/check/periodic-data-plane-adoption-osp-17-to-extracted-crc/f7c0a71/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/crs/openstackversions.core.openstack.org/openstack.yaml

Reply:
hey @marios, I suspect the problem is that manila and cinder-volume should match the instance name, otherwise the default container is used. By matching the instance name I mean that if there's a share called `share1`, you should have the `share1` entry in the openstackVersions CR, if you have a cinder-volume called `ceph`, you should have the same entry in the openstackVersions.

Reply:
thanks for looking @fmount

Reply:
However this is weird, as per [1] I would expect the `default` image there, so you don't need to explicitly define an entry. 
From `OpenStackVersions` logs [2] you can still see references of the `quay.io` related images, and I suspect the problem is that the current deployed version is 0.0.1, so the default images for cinder and manila are get from:

```
 containerImageVersionDefaults:
    0.0.1:
```

which I think we don't have in the current template.

[1] https://github.com/openstack-k8s-operators/openstack-operator/blob/main/pkg/openstack/manila.go#L123
[2] https://logserver.rdoproject.org/95/52995/2/check/periodic-data-plane-adoption-osp-17-to-extracted-crc/f7c0a71/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/crs/openstackversions.core.openstack.org/openstack.yaml

Reply:
thanks again for checking
we could easily test the version theory by including the version in the template (even hard-coded for the test) and if it works I can add a task to determine the current version to apply to the template.

I'd still push to merge this and I can iterate for this bit